<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>567</width>
    <height>366</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>567</width>
    <height>366</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>567</width>
    <height>366</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QWidget" name="horizontalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>160</y>
      <width>531</width>
      <height>102</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="cBtnLayout">
     <item>
      <widget class="QPushButton" name="mBtn">
       <property name="minimumSize">
        <size>
         <width>151</width>
         <height>81</height>
        </size>
       </property>
       <property name="text">
        <string>Set Scanning</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="lBtn">
       <property name="minimumSize">
        <size>
         <width>151</width>
         <height>81</height>
        </size>
       </property>
       <property name="text">
        <string>Set Movement</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="rBtn">
       <property name="minimumSize">
        <size>
         <width>151</width>
         <height>81</height>
        </size>
       </property>
       <property name="text">
        <string>Set Tactical</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>531</width>
      <height>102</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="imageLayout">
     <property name="spacing">
      <number>16</number>
     </property>
     <item>
      <widget class="QLabel" name="lLabel">
       <property name="minimumSize">
        <size>
         <width>100</width>
         <height>100</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>100</width>
         <height>100</height>
        </size>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="mLabel">
       <property name="minimumSize">
        <size>
         <width>100</width>
         <height>100</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>100</width>
         <height>100</height>
        </size>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="rLabel">
       <property name="minimumSize">
        <size>
         <width>100</width>
         <height>100</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>100</width>
         <height>100</height>
        </size>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget_3">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>270</y>
      <width>171</width>
      <height>63</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="turnLayout">
     <item>
      <widget class="QPushButton" name="gBtn">
       <property name="minimumSize">
        <size>
         <width>169</width>
         <height>61</height>
        </size>
       </property>
       <property name="text">
        <string>End Turn</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
